profiles/rawls-service-perimeters/terraform/rawls-service-perimeters/vpc-flow-egress-alert.tf

```diff
@@ -0,0 +1,157 @@
+  })
+  }
+}
+
+# Pull Sumologic credentials from Vault
+data "vault_generic_secret" "sumologic-secret-path" {
+  path = "secret/dsde/firecloud/dev/aou/sumologic"
+}
+
+provider "sumologic" {
+  access_id   = data.vault_generic_secret.sumologic-secret-path.data["access_id"]
+  access_key  = data.vault_generic_secret.sumologic-secret-path.data["access_key"]
+  environment = "us2"
+}
+
+# Creates the project to host Pub/Sub topic and subscription
+resource "google_project" "egress-alert-project" {
+  for_each              = var.vpc_flow_egress_alerts
+
+  name                  = replace("${each.key}-alerts", "_", "-")
+  project_id            = replace("${each.key}-alerts", "_", "-")
+  folder_id             = google_folder.folder[each.key].id
+}
+
+# A Pub/Sub topic to publish VPC flow logs
+resource "google_pubsub_topic" "vpc-flow-pubsub-topic" {
+  for_each   = var.vpc_flow_egress_alerts
+
+  project    = google_project.egress-alert-project[each.key].name
+  name       = replace("${each.key}-pubsub-topic", "_", "-")
+}
+
+# Log sink on AoU RW Service Perimeter folder and publish using Pub/Sub
+resource "google_logging_folder_sink" "vpc-flow-log-sink" {
+  for_each         = var.vpc_flow_egress_alerts
+
+  name             = "${each.key}-vpc-flow-log-sink"
+  folder           = google_folder.folder[each.key].name
+  destination      = "pubsub.googleapis.com/${google_pubsub_topic.vpc-flow-pubsub-topic[each.key].id}"
+  filter           = "logName:\"/logs/compute.googleapis.com%2Fvpc_flows\""
+  include_children = true
+}
+
+# The Sumologic collector
+resource "sumologic_collector" "vpc-flow-logs" {
+  name        = replace("terra-${var.terra_environment}-vpc-flow-log-collector", "_", "-")
+  description = "Sumologic collector"
+}
+
+# Sumologic GCP Source receives log data where the Pub/Sub message is published to.
+resource "sumologic_gcp_source" "sumologic-vpc-flow-log-source" {
+  for_each      = var.vpc_flow_egress_alerts
+
+  name          = "gcp/vpcflowlogs/aou/${lookup(lookup(var.vpc_flow_egress_alerts, each.key), "sumologic_source_category_name")}"
+  description   = "Sumologic GCP Source receives log data from Google Pub/Sub"
+  category      = "gcp"
+  collector_id  = sumologic_collector.vpc-flow-logs.id
+}
+
+# Subscribe flow log topic used by the Sumologic source above.
+resource "google_pubsub_subscription" "vpc-flow-pubsub-subscription" {
+  for_each   = var.vpc_flow_egress_alerts
+
+  project    = google_project.egress-alert-project[each.key].name
+  name       = replace("${each.key}-pubsub-sub", "_", "-")
+  topic      = google_pubsub_topic.vpc-flow-pubsub-topic[each.key].name
+
+  ack_deadline_seconds = 20
+
+  expiration_policy {
+    // No expiration
+    ttl = ""
+  }
+
+  push_config {
+    push_endpoint = sumologic_gcp_source.sumologic-vpc-flow-log-source[each.key].url
+
+    attributes = {
+      x-goog-version = "v1"
+    }
+  }
+}
+
+# Simply export a content file or folder and put the JSON file in ./assets/content.
```

Comment:
Not `./assets/content` anymore, we moved the files to the root dir since having stuff in sub-directories is incompatible with the bash wrapper templating stuff that makes this work.